app/entry.js

```diff
@@ -6,11 +6,56 @@ require('./scss/base.scss');
 
 // npm modules
```

Comment:
the layout of this module is great, it reads very well from top to bottom, and i can tell exactly what features of the app exist just from this one file :)!